<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orgNote</class>
 <widget class="QMainWindow" name="orgNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>563</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>563</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Org Note</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dataRes.qrc">
    <normaloff>:/images/images/main-icon.png</normaloff>:/images/images/main-icon.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>626</width>
      <height>497</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="centralGridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item row="3" column="1">
      <spacer name="secondVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>35</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="secondHorizontalLayout">
       <item>
        <widget class="QLabel" name="noteLabel">
         <property name="text">
          <string>N&amp;ote</string>
         </property>
         <property name="buddy">
          <cstring>noteEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="noteEdit">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>330</height>
          </size>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="autoFormatting">
          <set>QTextEdit::AutoBulletList</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <spacer name="firstVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QFrame" name="buttonFrame">
       <property name="minimumSize">
        <size>
         <width>521</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="hSpacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="statusTip">
             <string>Save the entry with the current information.</string>
            </property>
            <property name="text">
             <string> &amp;Save</string>
            </property>
            <property name="icon">
             <iconset resource="dataRes.qrc">
              <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="statusTip">
             <string>Search for an entry by the name entered OR|AND the tag.</string>
            </property>
            <property name="text">
             <string> Se&amp;arch</string>
            </property>
            <property name="icon">
             <iconset resource="dataRes.qrc">
              <normaloff>:/images/images/search-icon.png</normaloff>:/images/images/search-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="statusTip">
             <string>Delete the specified entry.</string>
            </property>
            <property name="text">
             <string> &amp;Delete</string>
            </property>
            <property name="icon">
             <iconset resource="dataRes.qrc">
              <normaloff>:/images/images/Delete-icon.png</normaloff>:/images/images/Delete-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="exitButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="statusTip">
             <string>Exit App.</string>
            </property>
            <property name="text">
             <string> E&amp;xit</string>
            </property>
            <property name="icon">
             <iconset resource="dataRes.qrc">
              <normaloff>:/images/images/log_out.png</normaloff>:/images/images/log_out.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="firstHorizontalLayout">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>&amp;Name</string>
         </property>
         <property name="buddy">
          <cstring>nameEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>24</height>
          </size>
         </property>
         <property name="statusTip">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="statusTip">
          <string>Clear all textboxes. (Ctrl + F.)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="dataRes.qrc">
           <normaloff>:/images/images/gnome_edit_clear.png</normaloff>:/images/images/gnome_edit_clear.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>19</width>
           <height>19</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+F</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="firstHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="tagLabel">
         <property name="text">
          <string>Ca&amp;tegory</string>
         </property>
         <property name="buddy">
          <cstring>tagComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tagComboBox">
         <property name="minimumSize">
          <size>
           <width>135</width>
           <height>24</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="maxVisibleItems">
          <number>6</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="googleButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="statusTip">
          <string>Google entry name with/or tag. (Ctrl + G)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="dataRes.qrc">
           <normaloff>:/images/images/icon-google.png</normaloff>:/images/images/icon-google.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>19</width>
           <height>19</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+G</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="statusTip">
    <string>Check this space for useful info.</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>noteEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>searchButton</tabstop>
 </tabstops>
 <resources>
  <include location="dataRes.qrc"/>
 </resources>
 <connections/>
</ui>
